src/index.js

```diff
@@ -148,3 +148,13 @@ export class TwoPartsLinks extends React.Component {
     return response;
   }
 }
+
+export class RotateText extends React.Component {
+  render() {
+    <a class="Rotate_text" href={this.props.href}>
```

Comment:
Add a return statement here! It does not work as it is not returning anything in the render method.

render() {
    return (....);
  }

Reply:
After this it's good to go! will merge.